prosemirror-lwdita/.eslintrc.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
I think for the most part this should be the same as https://github.com/evolvedbinary/lwdita/blob/main/.eslintrc
Although you don't need to add the `eslint-plugin-notice` in this PR